docs/docsite/rst/network_getting_started.rst

```diff
@@ -0,0 +1,246 @@
+.. network-getting-started-example:
+
+*******************************
+Network getting started example
+*******************************
+
+.. contents:: Topics
+
+
+Overview
+========
+
+Objective
+---------
+
+I want to understand how Ansible can be used to connect to multiple network devices.
+
+.. FIXME FUTURE Gundalow - Link to examples index once created
+
+.. note:: Examples, not production code
+   This is not for production use, it's more to demonstrate the different areas that need to be considered and provide a foundation level knowledge.
+
+Prerequisites
+-------------
+
+* Ansible 2.3 (or higher) installed :doc:`intro_installation`
+* One or more network device compatible with Ansible
+* Basic understanding of YAML :doc:`YAMLSyntax`
+* Basic understanding of Jinja2 Templates :doc:`playbooks_templating`
+
+.. FIXME FUTURE Gundalow - Once created we will link to the connection table here (which platforms support network_cli & credentials through inventory)
+.. FIXME FUTURE Gundalow -  Using ``ansible_ssh_pass`` will not work for REST transports such as ``eapi``, ``nxapi`` - Once documented in above FIXME add details her
+
+Audience
+--------
+
+* Basic Linux (comfortable on the command line)
```

Comment:
I guess this needs to go in the prerequisite section.  Audience,  would be net_admin or sys_admin, or any one who wants to play around with Ansible and networking.